data-source/jsonnet/england-wales/ccs/blocks/individual/confirm_dob.jsonnet

```diff
@@ -64,13 +64,13 @@ local proxyTitle = {
   ],
 };
 local proxyYesLabel = {
-  text: 'Yes, {person_name} is {age_in_years} years old',
+  text: 'Yes',
```

Comment:
Placeholder is no longer used so this variable can be removed and just added to the structure directly.